src/jailer/src/env.rs

```diff
@@ -71,21 +72,13 @@ impl Env {
         start_time_us: u64,
         start_time_cpu_us: u64,
     ) -> Result<Self> {
-        // All arguments are either mandatory, or have default values, so the unwraps
-        // should not fail.
+        // Mandatory arguments or with default values, unwraps should not fail.
```

Comment:
I changed it because now not `All arguments are either mandatory, or have default values`. That's why I added the `// Optional arguments` section below.